src/test/java/car/domain/CarTest.java

```diff
@@ -0,0 +1,33 @@
+package car.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CarTest {
+
+
+    @ParameterizedTest
+    @ValueSource(strings = {"abcdef"})
+    void 자동차_이름은_5자리를_초과하지않음(String strings) {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> new Car(strings));
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = 10)
+    void 원하는_수_사이의_랜덤_수를_생성한다(Integer ints) {
+        List<Integer> number = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9);
+        Boolean numberCheck = number.contains(RandomNumber.getRandomNumber(ints));
+        assertThat(numberCheck).isTrue();
```

Comment:
테스트를 작성한다면 아래와 같이 검증하는 방법도 가능할 것으로 보이네요.
```suggestion
        assertThat(numberCheck)
            .isGreaterThan(0)
            .isLessThan(10);
```

다만, 이 테스트는 사실상 `Random` 클래스에서 제공해주는 난수가 잘 만들어지는지에 대한 테스트인데, 이는 자바 라이브러리에 대한 테스트로 보입니다. 우리가 테스트하고 싶은 부분이 `Random` 클래스의 정상 동작 여부인지 고민해 볼 필요가 있겠습니다.